### DISCUSSION TITLE
422 Unprocessable Entity error after converted by Cython

### DISCUSSION THREAD

**Discussion by ambrosekwok** [fastapi/fastapi#7272] [Questions]
This is my python code app.py:
```Python
import uvicorn
from fastapi import FastAPI
from pydantic import BaseModel

app = FastAPI()

class Input(BaseModel):
    data1: str
    data2: int
    data3: float

@app.put('/test')
def test(args: Input):
    results = {
        "data1": args.data1,
        "data2": args.data2,
        "data3": args.data3
    }
    return results

if __name__ == "__main__":
    uvicorn.run(app, host="0.0.0.0", port=8000)
```

This is the Bash command to convert above app.py to executable and run
```Bash
cython --directive always_allow_keywords=true --embed -3 -o app.c app.py
gcc $(pkg-config --cflags python3) app.c $(pkg-config python3-embed --libs || pkg-config python3 --libs) -o app
sh -c ./app
```

### Description
After converted to executable, request body content missing:
![image](https://user-images.githubusercontent.com/37961681/91930255-f727db00-ed12-11ea-9f12-d67f3c942e86.png)

If manual add data into request body, this error return.
```Bash
INFO:     127.0.0.1:59896 - "PUT /test HTTP/1.1" 422 Unprocessable Entity
```
![image](https://user-images.githubusercontent.com/37961681/91930445-6ef60580-ed13-11ea-83b9-48293eedf0db.png)

Run python directly work as expected:
![image](https://user-images.githubusercontent.com/37961681/91930203-cc3d8700-ed12-11ea-95a2-bde5682aadbf.png)

### Environment

* OS: Linux: Ubuntu 20.04
* FastAPI Version: 0.61.1
* Python version: 3.8.2

Thank you.

Ambrose

**Comment by tiangolo** [comment]
I imagine you solved your problem, so thanks for coming back to close the issue! ☕ 

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
